extras/common/app/views/kaminari/blacklight/_prev_page.html.erb

```diff
@@ -0,0 +1,27 @@
+<%# Link to the "Previous" page
```

Comment:
Added this file in order to put `disabled` attribute on pagination Previous link (CSS class .disabled will not suffice).